custom/bin/ARE/data/webservice/angular/envControl/component/help/faq/html/faq6en.html

```diff
@@ -1,14 +1,14 @@
-<p>In der Regel wird zur Bedienung der AsTeRICS Ergo Umgebungssteuerung dasselbe System wie zur Steuerung
-    des Computers verwendet.
+<p>Usually you would use the same system for controlling AsTeRICS Ergo as you control your computer. This can be done using a conventional mouse or an alternative computer input device.
 </p>
 <p>Bedienungsmöglichkeiten:
```

Comment:
this line still needs to be translated. (I could do it on my own, but I do a comment now, so that you see how commenting & discussing changes works here on github) :) So you can just edit the line using the pencil above and I will get a notification and then will merge the branch.